### PR TITLE
Improve action creators - Part 1

### LINKED ISSUE #414: Improve action creators

### PR DESCRIPTION
**Pull request by v08uka**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/EventsExpress/EventsExpress/projects/2)

## Code reviewers

- [ ] @iuriikhrystiuk 
e
### Second Level Review

- [x] @sand0 

## CHECK LIST
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

**Comment by v08uka** [comment]
About code smells:
I didn't fix it in reducers, because this is by reduser conventional.
And "Update identical function" because this is an old version and will be removed after a complete refactoring.

File: EventsExpress/ClientApp/src/actions/eventSchedule-item-view-action.js [hunk 1]
```diff
@@ -1,27 +1,23 @@
...
-            if (response.error == null) {
-                dispatch(get_eventSchedule(response));
-            } else {
-                dispatch(getEventScheduleError(response.error));
-            }
-        });
+        let response = await api_serv.getEventSchedule(id);
+        if (!response.ok) {
+            dispatch(setErrorAllertFromResponse(response));
+            return Promise.resolve();
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/actions/eventSchedule-item-view-action.js]: Maybe we need to do promise.reject here and promise.resolve below?

File: EventsExpress/ClientApp/src/actions/logout-action.js [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
-import { updateEventsFilters } from './event-list';
+import { updateEventsFilters } from './event-list-action';
 
 export const SET_LOGOUT = "SET_LOGOUT";
 
+const api_serv = new AuthenticationService();
+
 export default function logout() {
-    revokeToken();
+    api_serv.revokeToken();
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/actions/logout-action.js]: Do we have to do `await` here?

Comment (sand0) [reply EventsExpress/ClientApp/src/actions/logout-action.js]: Actually, we will not do any things with the response, this is only background work, so we can skip `await` to force accelerate the next invocations

**Review by sand0 — APPROVED** [review]

**Pull request merged**